### ISSUE TITLE
2025-05-21, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#176]
# Today's Arxiv Papers

(1) [Mind the Gap: Bridging Thought Leap for Improved Chain-of-Thought Tuning](http://arxiv.org/abs/2505.14684v1), 2025-05-20

(2) [UniGen: Enhanced Training & Test-Time Strategies for Unified Multimodal Understanding and Generation](http://arxiv.org/abs/2505.14682v1), 2025-05-20

(3) [UltraEdit: Training-, Subject-, and Memory-Free Lifelong Editing in Large Language Models](http://arxiv.org/abs/2505.14679v1), 2025-05-20

(4) [Visionary-R1: Mitigating Shortcuts in Visual Reasoning with Reinforcement Learning](http://arxiv.org/abs/2505.14677v1), 2025-05-20

(5) [Reward Reasoning Model](http://arxiv.org/abs/2505.14674v1), 2025-05-20

(6) [Quartet: Native FP4 Training Can Be Optimal for Large Language Models](http://arxiv.org/abs/2505.14669v1), 2025-05-20

(7) [ContextAgent: Context-Aware Proactive LLM Agents with Open-World Sensory Perceptions](http://arxiv.org/abs/2505.14668v1), 2025-05-20

(8) [SAFEPATH: Preventing Harmful Reasoning in Chain-of-Thought via Early Alignment](http://arxiv.org/abs/2505.14667v1), 2025-05-20

(9) [Abacus: A Cost-Based Optimizer for Semantic Operator Systems](http://arxiv.org/abs/2505.14661v1), 2025-05-20

(10) [Cost-Augmented Monte Carlo Tree Search for LLM-Assisted Planning](http://arxiv.org/abs/2505.14656v1), 2025-05-20

(11) [Beyond Words: Multimodal LLM Knows When to Speak](http://arxiv.org/abs/2505.14654v1), 2025-05-20

(12) [General-Reasoner: Advancing LLM Reasoning Across All Domains](http://arxiv.org/abs/2505.14652v1), 2025-05-20

(13) [Think Only When You Need with Large Hybrid-Reasoning Models](http://arxiv.org/abs/2505.14631v1), 2025-05-20

(14) [KERL: Knowledge-Enhanced Personalized Recipe Recommendation using Large Language Models](http://arxiv.org/abs/2505.14629v1), 2025-05-20

(15) [Debating for Better Reasoning: An Unsupervised Multimodal Approach](http://arxiv.org/abs/2505.14627v1), 2025-05-20

(16) [TinyV: Reducing False Negatives in Verification Improves RL for LLM Reasoning](http://arxiv.org/abs/2505.14625v1), 2025-05-20

(17) [Enhancing Learned Knowledge in LoRA Adapters Through Efficient Contrastive Decoding on Ascend NPUs](http://arxiv.org/abs/2505.14620v1), 2025-05-20

(18) [Linear Control of Test Awareness Reveals Differential Compliance in Reasoning Models](http://arxiv.org/abs/2505.14617v1), 2025-05-20

(19) [SATBench: Benchmarking LLMs' Logical Reasoning via Automated Puzzle Generation from SAT Formulas](http://arxiv.org/abs/2505.14615v1), 2025-05-20

(20) [sudoLLM : On Multi-role Alignment of Language Models](http://arxiv.org/abs/2505.14607v1), 2025-05-20

(21) [Let LLMs Break Free from Overthinking via Self-Braking Tuning](http://arxiv.org/abs/2505.14604v1), 2025-05-20

(22) [Toward Reliable Biomedical Hypothesis Generation: Evaluating Truthfulness and Hallucination in Large Language Models](http://arxiv.org/abs/2505.14599v1), 2025-05-20

(23) [Success is in the Details: Evaluate and Enhance Details Sensitivity of Code LLMs through Counterfactuals](http://arxiv.org/abs/2505.14597v1), 2025-05-20

(24) [MCIP: Protecting MCP Safety via Model Contextual Integrity Protocol](http://arxiv.org/abs/2505.14590v1), 2025-05-20

(25) [Context Reasoner: Incentivizing Reasoning Capability for Contextualized Privacy and Safety Compliance via Reinforcement Learning](http://arxiv.org/abs/2505.14585v1), 2025-05-20

(26) [Can Pruning Improve Reasoning? Revisiting Long-CoT Compression with Capability in Mind for Better Reasoning](http://arxiv.org/abs/2505.14582v1), 2025-05-20

(27) [TRATES: Trait-Specific Rubric-Assisted Cross-Prompt Essay Scoring](http://arxiv.org/abs/2505.14577v1), 2025-05-20

(28) [KORGym: A Dynamic Game Platform for LLM Reasoning Evaluation](http://arxiv.org/abs/2505.14552v1), 2025-05-20

(29) [Can Large Language Models Really Recognize Your Name?](http://arxiv.org/abs/2505.14549v1), 2025-05-20

(30) [Breaking Bad Tokens: Detoxification of LLMs Using Sparse Autoencoders](http://arxiv.org/abs/2505.14536v1), 2025-05-20

(31) [Internal Chain-of-Thought: Empirical Evidence for Layer-wise Subtask Scheduling in LLMs](http://arxiv.org/abs/2505.14530v1), 2025-05-20

(32) [BugRepro: Enhancing Android Bug Reproduction with Domain-Specific Knowledge Integration](http://arxiv.org/abs/2505.14528v1), 2025-05-20

(33) [Guarded Query Routing for Large Language Models](http://arxiv.org/abs/2505.14524v1), 2025-05-20

(34) [Teaching Audio-Aware Large Language Models What Does Not Hear: Mitigating Hallucinations through Synthesized Negative Samples](http://arxiv.org/abs/2505.14518v1), 2025-05-20

(35) [Latent Flow Transformer](http://arxiv.org/abs/2505.14513v1), 2025-05-20

(36) [ModRWKV: Transformer Multimodality in Linear Time](http://arxiv.org/abs/2505.14505v1), 2025-05-20

(37) [Enhanced Multimodal Aspect-Based Sentiment Analysis by LLM-Generated Rationales](http://arxiv.org/abs/2505.14499v1), 2025-05-20

(38) [Reasoning Models Better Express Their Confidence](http://arxiv.org/abs/2505.14489v1), 2025-05-20

(39) [MoMoE: Mixture of Moderation Experts Framework for AI-Assisted Online Governance](http://arxiv.org/abs/2505.14483v1), 2025-05-20

(40) [Towards Reliable Proof Generation with LLMs: A Neuro-Symbolic Approach](http://arxiv.org/abs/2505.14479v1), 2025-05-20

(41) [Adapting Pretrained Language Models for Citation Classification via Self-Supervised Contrastive Learning](http://arxiv.org/abs/2505.14471v1), 2025-05-20

(42) [Attributional Safety Failures in Large Language Models under Code-Mixed Perturbations](http://arxiv.org/abs/2505.14469v1), 2025-05-20

(43) [ServerlessLoRA: Minimizing Latency and Cost in Serverless Inference for LoRA-Based LLMs](http://arxiv.org/abs/2505.14468v1), 2025-05-20

(44) [VisualQuality-R1: Reasoning-Induced Image Quality Assessment via Reinforcement Learning to Rank](http://arxiv.org/abs/2505.14460v1), 2025-05-20

(45) [Video Compression Commander: Plug-and-Play Inference Acceleration for Video Large Language Models](http://arxiv.org/abs/2505.14454v1), 2025-05-20

(46) [Creative Preference Optimization](http://arxiv.org/abs/2505.14442v1), 2025-05-20

(47) [S2SBench: A Benchmark for Quantifying Intelligence Degradation in Speech-to-Speech Large Language Models](http://arxiv.org/abs/2505.14438v1), 2025-05-20

(48) [Neural Incompatibility: The Unbridgeable Gap of Cross-Scale Parametric Knowledge Transfer in Large Language Models](http://arxiv.org/abs/2505.14436v1), 2025-05-20

(49) [Choosing a Model, Shaping a Future: Comparing LLM Perspectives on Sustainability and its Relationship with AI](http://arxiv.org/abs/2505.14435v1), 2025-05-20

(50) [Rank-K: Test-Time Reasoning for Listwise Reranking](http://arxiv.org/abs/2505.14432v1), 2025-05-20

(51) [SkyMemory: A LEO Edge Cache for Transformer Inference Optimization and Scale Out](http://arxiv.org/abs/2505.14427v1), 2025-05-20

(52) [From Templates to Natural Language: Generalization Challenges in Instruction-Tuned LLMs for Spatial Reasoning](http://arxiv.org/abs/2505.14425v1), 2025-05-20

(53) [Scaling Low-Resource MT via Synthetic Data Generation with LLMs](http://arxiv.org/abs/2505.14423v1), 2025-05-20

(54) [MindVote: How LLMs Predict Human Decision-Making in Social Media Polls](http://arxiv.org/abs/2505.14422v1), 2025-05-20

(55) [Hidden Ghost Hand: Unveiling Backdoor Vulnerabilities in MLLM-Powered Mobile GUI Agents](http://arxiv.org/abs/2505.14418v1), 2025-05-20

(56) [Towards Non-Euclidean Foundation Models: Advancing AI Beyond Euclidean Frameworks](http://dx.doi.org/10.1145/3701716.3717806), 2025-05-20

(57) [PRL: Prompts from Reinforcement Learning](http://arxiv.org/abs/2505.14412v1), 2025-05-20

(58) [Pierce the Mists, Greet the Sky: Decipher Knowledge Overshadowing via Knowledge Circuit Analysis](http://arxiv.org/abs/2505.14406v1), 2025-05-20

(59) [Unearthing Gems from Stones: Policy Optimization with Negative Sample Augmentation for LLM Reasoning](http://arxiv.org/abs/2505.14403v1), 2025-05-20

(60) [Log-Augmented Generation: Scaling Test-Time Reasoning with Reusable Computation](http://arxiv.org/abs/2505.14398v1), 2025-05-20

(61) [Causal Cartographer: From Mapping to Reasoning Over Counterfactual Worlds](http://arxiv.org/abs/2505.14396v1), 2025-05-20

(62) [MUG-Eval: A Proxy Evaluation Framework for Multilingual Generation Capabilities in Any Language](http://arxiv.org/abs/2505.14395v1), 2025-05-20

(63) [Knowledge Graph Based Repository-Level Code Generation](http://arxiv.org/abs/2505.14394v1), 2025-05-20

(64) [Editing Across Languages: A Survey of Multilingual Knowledge Editing](http://arxiv.org/abs/2505.14393v1), 2025-05-20

(65) [Beyond the First Error: Process Reward Models for Reflective Mathematical Reasoning](http://arxiv.org/abs/2505.14391v1), 2025-05-20

(66) [SCAN: Semantic Document Layout Analysis for Textual and Visual Retrieval-Augmented Generation](http://arxiv.org/abs/2505.14381v1), 2025-05-20

(67) [AutoRev: Automatic Peer Review System for Academic Research Papers](http://arxiv.org/abs/2505.14376v1), 2025-05-20

(68) [Is Your Prompt Safe? Investigating Prompt Injection Attacks Against Open-Source LLMs](http://arxiv.org/abs/2505.14368v1), 2025-05-20

(69) [Dual Decomposition of Weights and Singular Value Low Rank Adaptation](http://arxiv.org/abs/2505.14367v1), 2025-05-20

(70) [PersonaTAB: Predicting Personality Traits using Textual, Acoustic, and Behavioral Cues in Fully-Duplex Speech Dialogs](http://arxiv.org/abs/2505.14356v1), 2025-05-20

(71) [WirelessMathBench: A Mathematical Modeling Benchmark for LLMs in Wireless Communications](http://arxiv.org/abs/2505.14354v1), 2025-05-20

(72) [Towards eliciting latent knowledge from LLMs with mechanistic interpretability](http://arxiv.org/abs/2505.14352v1), 2025-05-20

(73) [OSoRA: Output-Dimension and Singular-Value Initialized Low-Rank Adaptation](http://arxiv.org/abs/2505.14350v1), 2025-05-20

(74) [QA-prompting: Improving Summarization with Large Language Models using Question-Answering](http://arxiv.org/abs/2505.14347v1), 2025-05-20

(75) [Scaling and Enhancing LLM-based AVSR: A Sparse Mixture of Projectors Approach](http://arxiv.org/abs/2505.14336v1), 2025-05-20

(76) [Breaking Down Video LLM Benchmarks: Knowledge, Spatial Perception, or True Temporal Understanding?](http://arxiv.org/abs/2505.14321v1), 2025-05-20

(77) [RADAR: Enhancing Radiology Report Generation with Supplementary Knowledge Injection](http://arxiv.org/abs/2505.14318v1), 2025-05-20

(78) [Exploring Jailbreak Attacks on LLMs through Intent Concealment and Diversion](http://arxiv.org/abs/2505.14316v1), 2025-05-20

(79) [Low-Cost FlashAttention with Fused Exponential and Multiplication Hardware Operators](http://arxiv.org/abs/2505.14314v1), 2025-05-20

(80) [A MIND for Reasoning: Meta-learning for In-context Deduction](http://arxiv.org/abs/2505.14313v1), 2025-05-20

(81) [HausaNLP: Current Status, Challenges and Future Directions for Hausa Natural Language Processing](http://arxiv.org/abs/2505.14311v1), 2025-05-20

(82) [JOLT-SQL: Joint Loss Tuning of Text-to-SQL with Confusion-aware Noisy Schema Sampling](http://arxiv.org/abs/2505.14305v1), 2025-05-20

(83) [Scaling Law for Quantization-Aware Training](http://arxiv.org/abs/2505.14302v1), 2025-05-20

(84) [SafetyNet: Detecting Harmful Outputs in LLMs by Modeling and Monitoring Deceptive Behaviors](http://arxiv.org/abs/2505.14300v1), 2025-05-20

(85) [Empowering LLMs in Task-Oriented Dialogues: A Domain-Independent Multi-Agent Framework and Fine-Tuning Strategy](http://arxiv.org/abs/2505.14299v1), 2025-05-20

(86) [Cross-Lingual Optimization for Language Transfer in Large Language Models](http://arxiv.org/abs/2505.14297v1), 2025-05-20

(87) [Universal Acoustic Adversarial Attacks for Flexible Control of Speech-LLMs](http://arxiv.org/abs/2505.14286v1), 2025-05-20

(88) [YESciEval: Robust LLM-as-a-Judge for Scientific Question Answering](http://arxiv.org/abs/2505.14279v1), 2025-05-20

(89) [Think-J: Learning to Think for Generative LLM-as-a-Judge](http://arxiv.org/abs/2505.14268v1), 2025-05-20

(90) [AAPO: Enhance the Reasoning Capabilities of LLMs with Advantage Momentum](http://arxiv.org/abs/2505.14264v1), 2025-05-20

(91) [Speculative Decoding Reimagined for Multimodal Large Language Models](http://arxiv.org/abs/2505.14260v1), 2025-05-20

(92) [FuxiMT: Sparsifying Large Language Models for Chinese-Centric Multilingual Machine Translation](http://arxiv.org/abs/2505.14256v1), 2025-05-20

(93) [TransBench: Benchmarking Machine Translation for Industrial-Scale Applications](http://arxiv.org/abs/2505.14244v1), 2025-05-20

(94) [ABBA: Highly Expressive Hadamard Product Adaptation for Large Language Models](http://arxiv.org/abs/2505.14238v1), 2025-05-20

(95) [UniVG-R1: Reasoning Guided Universal Visual Grounding with Reinforcement Learning](http://arxiv.org/abs/2505.14231v1), 2025-05-20

(96) ["Haet Bhasha aur Diskrimineshun": Phonetic Perturbations in Code-Mixed Hinglish to Red-Team LLMs](http://arxiv.org/abs/2505.14226v1), 2025-05-20

(97) [Reinforcement Learning vs. Distillation: Understanding Accuracy and Capability in LLM Reasoning](http://arxiv.org/abs/2505.14216v1), 2025-05-20

(98) [Safety Devolution in AI Agents](http://arxiv.org/abs/2505.14215v1), 2025-05-20

(99) [Automatic Dataset Generation for Knowledge Intensive Question Answering Tasks](http://arxiv.org/abs/2505.14212v1), 2025-05-20

(100) [Capturing the Effects of Quantization on Trojans in Code LLMs](http://arxiv.org/abs/2505.14200v1), 2025-05-20
